smarts/sstudio/genhistories.py

```diff
@@ -803,6 +803,10 @@ def _check_args(args) -> bool:
         with open(args.dataset, "r") as yf:
             dataset_spec = yaml.safe_load(yf)["trajectory_dataset"]
 
+    if dataset_spec.get("input_path") == "<PATH TO FILE GOES HERE>":
+        print(f"skipping placeholder dataset spec at {args.dataset}.")
+        sys.exit(0)
```

Comment:
I believe we should be more aggressive than this. The dataset is not useable without "input_path".

Reply:
This was added to effectively skip these scenarios in our CI, since we won't be pushing the datasets in our repo.  (These scenarios can only be built by a user who has downloaded the dataset to their local filesystem and then set their `input_path` accordingly.)  So it has to exit with a status of 0, otherwise CI will think that `scl scenario build-all` failed.  

What did you have in mind here?

Reply:
Ah, I see. I think my preference would be to exclude these scenarios from the `build-all`.

It is more of a nitpick than a requirement though.